### DISCUSSION TITLE
How do you want to find out about updates?

### DISCUSSION THREAD

**Discussion by briandfoy** [briandfoy/cpan-audit#8] [Polls]
What's the best way to tell you there's updated data?

There is an RSS Feed for github releases: https://github.com/briandfoy/cpan-audit/releases.atom

You can track CPAN Changes files with newsreleases.io. Look for the "Perl CPAN" source, then for the "Project Name or URL", use the URL for the Changes file in MetaCPAN, such as https://metacpan.org/dist/CPAN-Audit/changes

**Comment by n1vux** [comment]
Choice is good. TIMTOWTDI !

**Reply by briandfoy** [reply]
Sure, but what you do actually use, especially if I don't already have a way to provide it? Perhaps vote for the one you prefer overall. :)
